pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -459,6 +471,41 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	}
 	//#endregion
 
+	//#region Backups
```

Comment:
This doesn't yet do anything to reconcile the backups after the first creation, which it really should (e.g. someone changing the schedule for a backup should cause it to update (rather than them having to update the generated CronJob, or worse, having to delete and recreate the UnifiedPushServer CR). Similarly adding/removing backup entries should cause the appropriate CronJobs to be added/removed.

I haven't added that yet, because there's a regression here in our monstrous reconcile function: once the `status.phase` is set to `Complete`, we ignore any other changes (both on the cluster and to the CR). This means that even if someone deleted the UPS or PostgreSQL deployments, we wouldn't be recreating them, which is not what we want. I'll create another issue for that and link it here (and also mention that then updates to backup entries or the resulting CronJobs should be handled appropriately).

Reply:
You're right.

I added that check during my work about limiting the number of UPS servers to 1. But I see that this check I added isn't really required for that. Feel free to remove it.